addons/userconfig/cba_settings.sqf

```diff
@@ -73,11 +76,6 @@ force ace_medical_treatment_medicSurgicalKit = 1;         // Only medic can use
 force ace_medical_treatment_timeCoefficientPAK = 1.5;     // Time Coefficient that Modifies how long a PAK takes to apply
 force ace_medical_treatment_woundReopenChance = 1;        // Wound Reopening Coefficient
 
-force ace_nightvision_disableNVGsWithSights = false;
-force ace_nightvision_fogScaling = 1;
-force ace_nightvision_effectScaling = 1;
-force ace_nightvision_aimDownSightsBlur = 1;
```

Comment:
I suggest we leave these for brevity. It's easy to just link this file if anyone ask what our (default) settings are.

Reply:
Sure, if you want to 👍